src/api/init.ts

```diff
@@ -43,6 +43,9 @@ import { createStatusRouter } from './routes/status';
 import { createTokenRouter } from './routes/tokens/tokens';
 import { createFeeRateRouter } from './routes/fee-rate';
 import { setResponseNonCacheable } from './controllers/cache-controller';
+import * as swaggerUi from 'swagger-ui-express';
+import * as yaml from 'yamljs';
+const swaggerDocument = yaml.load('docs/openapi.yaml');
```

Comment:
`/doc` should not return the YAML document but the actual Redoc HTML generated documentation